lend-landing/src/components/HeroContent/VideoBackground.tsx

```diff
@@ -56,13 +51,46 @@ const Clip4 = styled.div`
 
 const VideoBackground: React.FC = () => {
   const { video } = useScreenSize();
+  const [clipCount, setCount] = useState<number>(0);
+  const [isLoading, setLoading] = useState<boolean>(true);
+  const clipAmount: number = 4;
+
+  const bg = useMemo(() => {
+    return (
+      <ReactPlayer
+        key="video-player"
+        url="https://player.vimeo.com/video/748213676?h=6ac437c1b3&amp;badge=0&amp;autopause=0&amp;player_id=0&amp;app_id=58479"
+        playing={true}
+        width={video.width}
+        height={video.height}
+        style={{
+          minWidth: video.width,
+          minHeight: video.height,
+          position: "absolute",
+        }}
+        muted
+        loop
+        onReady={() => {
+          setCount(clipCount + 1);
+        }}
+      />
+    );
+  }, [clipCount]);
+
+  useEffect(() => {
```

Comment:
It seems there isn't really an easy way to use a single instance of the video.. I tried making a custom <svg> that has the clip paths on it but it becomes quite complicated when you consider that you can't do CSS calculations (like your `inset`) in SVG format, not to mention mobile responsiveness as well.

I ditched it and used a very rugged approach by just counting the number of clips to await before proceeding to show all videos in one go

Reply:
yes same i thought i could do svg at the start but its too complicated.